packages/react-native-web/src/exports/Modal/ModalAnimation.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) Nicolas Gallagher.
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import { useEffect, useState, useCallback } from 'react';
+
+import StyleSheet from '../StyleSheet';
+import createElement from '../createElement';
+
+const ANIMATION_DURATION = 300;
+
+function getAnimationStyle(animationType, visible) {
+  if (animationType === 'slide') {
+    return visible ? animatedSlideInStyles : animatedSlideOutStyles;
+  }
+
+  if (animationType === 'fade') {
+    return visible ? animatedFadeInStyles : animatedFadeOutStyles;
+  }
+
+  return visible ? styles.container : styles.hidden;
+}
+
+export type ModalAnimationProps = {|
+  children?: any,
+
```

Comment:
Please remove the blank lines here. There are several other places where the formatting is like this. I won't comment on them all but please make the same change elsewhere